frontend/src/metabase/admin/upsells/UpsellCloud.tsx

```diff
@@ -0,0 +1,42 @@
+import { t } from "ttag";
+
+import { useHasTokenFeature } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+
+import { UpsellBigCard } from "./components";
+import S from "./components/Upsells.module.css";
+
+export const UpsellCloud = ({
+  source,
+  onOpenModal,
+}: {
+  source: string;
+  onOpenModal: () => void;
+}) => {
+  const isHosted = useHasTokenFeature("hosting");
+
+  if (isHosted) {
+    return null;
+  }
+
+  return (
+    <UpsellBigCard
+      title={t`Migrate to Metabase Cloud`}
+      campaign="cloud"
+      buttonText={t`Try for free`}
+      source={source}
+      onOpenModal={onOpenModal}
+      illustrationSrc="app/assets/img/upsell-cloud.png"
+    >
+      {t`Get automatic backups, restores, and upgrades, built-in network monitoring, unlimited expert help from engineers and more.`}{" "}
```

Comment:
"...restores, and upgrades..." Though these look a lot like verbs, they're nouns. Could we add a contextual note to point this out to the translators?

Reply:
77329aa9e33a84c22b0bd34db84de7118af35075